packages/Poll/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export default class Poll extends React.Component {
```

Comment:
What do you think about renaming to `<Interval />`? Poll, to me, implies fetching data in intervals, but this is more generic, yeah?

Reply:
agreed